test/services/implementations/elasticsearch.test.js

```diff
@@ -201,7 +201,8 @@ describe('Test: ElasticSearch service', () => {
           should(result).be.an.Object();
           should(result.total).be.exactly(0);
           should(result.hits).be.an.Array();
-          should(result._scroll_id).be.an.exactly('banana42');
+          should(result._scroll_id).be.exactly('banana42');
```

Comment:
I think you forgot to rename this scroll ID

Reply:
Nope, I did not. Check the line just below the one you quoted.

Reply:
Oh ok my bad :smile: